docs/syncset.md

```diff
@@ -41,6 +41,18 @@ spec:
     patch: |-
       { "data": { "foo": "new-bar" } }
     patchType: merge
+    
+  secretReferences:
+  - source:
+      kind: Secret
```

Comment:
The kind and apiVersion can be omitted in both the source and the target.

Reply:
Yeah those fields would be unused here and they should probably be removed.

Reply:
cool, removed those